AElf.Consensus/IConsensusService.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AElf.Common;
+using AElf.Kernel;
+
+namespace AElf.Consensus
+{
+    public interface IConsensusService
+    {
+        Task TriggerConsensus(int chainId);
+        Task StopConsensus();
```

Comment:
do not start/stop in service layer. if it is a service, we should not care about the service's state, it means we should not have start/stop method